neqo-http3/Cargo.toml

```diff
@@ -20,3 +20,4 @@ test-fixture = { path = "../test-fixture" }
 [features]
 default = ["deny-warnings"]
 deny-warnings = []
+fuzzing_h = ["neqo-transport/fuzzing_t", "neqo-crypto/fuzzing"]
```

Comment:
why the different name?

Reply:
I got confuse reading rust rules. The same name is allowed here. There was something else that does not allow same name.